src/main.rs

```diff
@@ -204,6 +204,11 @@ fn setup(args: &[String]) -> Setup {
             "",
             "disable-gapless",
             "disable gapless playback.",
+        )
+	    .optflag(
+            "",
+            "passthrough",
+            "Pass raw stream to output, only works for \"pipe\"."
```

Comment:
I might overlook something, but shouldn't we check that the backend is indeed pipe if this option is present?

Reply:
Or, we could even force the backend to be pipe - that was my initial opinion

Reply:
I prefer an error than forcing the backend. It would be less surprising to users.

Reply:
ok